roles/azimuth_capi_operator/defaults/main.yml

```diff
@@ -645,6 +645,79 @@ azimuth_capi_operator_app_templates_jupyterhub_spec:
         )
     }}
 
+# Indicates whether the Zenith and Intel XPU enabled JupyterHub app template should be enabled
+azimuth_capi_operator_app_templates_jupyterhub_intel_enabled: false
+# The Helm chart to use for the Intel XPU enabled JupyterHub app template
+azimuth_capi_operator_app_templates_jupyterhub_intel_chart_name: jupyterhub-azimuth
+# The Helm repository to use for the Intel XPU enabled JupyterHub app template
+azimuth_capi_operator_app_templates_jupyterhub_intel_chart_repo: https://azimuth-cloud.github.io/azimuth-charts
+# The version range to use for the Intel XPU enabled JupyterHub Helm chart
+azimuth_capi_operator_app_templates_jupyterhub_intel_version_range: ">=0.0.0"
+# Overrides for the label, logo and description from the Intel XPU enabled JupyterHub Helm chart
+azimuth_capi_operator_app_templates_jupyterhub_intel_label: JupyterHub with Intel GPUs
+azimuth_capi_operator_app_templates_jupyterhub_intel_logo:
+azimuth_capi_operator_app_templates_jupyterhub_intel_description: Multi-user Jupyter notebook environment with Intel GPU support.
+# The number of versions to make available for the Intel XPU enabled JupyterHub app template
+azimuth_capi_operator_app_templates_jupyterhub_intel_keep_versions: 3
+# The sync frequency for the Intel XPU enabled JupyterHub app template (default 24h)
+azimuth_capi_operator_app_templates_jupyterhub_intel_sync_frequency: 86400
+# Any default values for the Intel XPU enabled JupyterHub app template
+azimuth_capi_operator_app_templates_jupyterhub_intel_default_values:
+  jupyterhub:
+    singleuser:
+      image:
+        name: ghcr.io/johngarbutt/jupyterhub-intel-gpu
```

Comment:
Actually, we probably shouldn't merge this one, lets add it as extra config where this is wanted.